docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+
+To create a new F# project, open the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+```
+>f#: New Project
+```
+
+This is powered by the [FORGE](https://github.com/fsprojects/Forge) project.  You should see something like this:
+
+![](media/getting-started-vscode/vscode-new-proj.png)
+
+Select "F#: New Project" by hitting **Enter**, which will take you to this step:
+
+![](media/getting-started-vscode/vscode-proj-type.png)
+
+This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](http://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach this step:
+
+![](media/getting-started-vscode/vscode-new-dir.png)
+
+Here you can choose to create the project in a specific directory.  If you leave it blank, it will choose the current directory Visual Studio Code is opened in.  This article uses the current directory.  Once you've made your decision, hit **Enter** to bring up the final stage:
+
+![](media/getting-started-vscode/vscode-proj-name.png)
+
+This lets you name your project.  F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names.  This article uses `ClassLibrarySample` as the name.  Once you've entered the name you want for your project, hit **Enter**.
+
+If you followed these steps, you should get the Explorer on the left-hand side to look something like this:
+
+![](media/getting-started-vscode/vscode-new-proj-explorer.png)
+
+This template generates a few things you'll find useful:
+
+1. The F# project itself, underneath the `ClassLibraryDemo` folder.
+2. The correct directory structure for adding packages via [`Paket`](http://fsprojects.github.io/Paket/).
+3. A cross-platform build script with [`FAKE`](http://fsharp.github.io/FAKE/).
+4. The `paket.exe` executable which can fetch packages and resolve dependencies for you.
+5. A `.gitignore` file if you wish to add this project to Git-based source control.
+
+## Writing some code
+
+Open the *ClassLibraryDemo* folder.  You should see the following files:
+
+1. `ClassLibraryDemo.fs`, an F# source file with a class defined.
+2. `CassLibraryDemo.fsproj`, an F# project file used to build this project.
+3. `Script.fsx`, an F# script file which loads the source file.
+4. `paket.references`, a Paket file which specifies the project dependencies.
+
+Open `Script.fsx`, and add the following code at the end of it:
+
+[!code-fsharp[ToPigLatin](../../../../samples/snippets/fsharp/getting-started/to-pig-latin.fsx)]
+
+This function converts a word to a form of [Pig Latin](https://en.wikipedia.org/wiki/Pig_Latin).  Don't worry about the details of the code - that will be covered in a [later section](getting-started-vscode#explaining-the-code).  The next step is to execute it using F# Interactive (FSI).
```

Comment:
```
[later section](getting-started-vscode#explaining-the-code)
```

Should this be `[later section](getting-started-vscode.md#explaining-the-code)`?